composer.json

```diff
@@ -30,7 +29,9 @@
     },
     "autoload": {
         "psr-4": {
-            "Pim\\Bundle\\IcecatConnectorBundle\\": "src/"
+            "Pim\\Bundle\\IcecatConnectorBundle\\": "src/Icecat/",
+            "Pim\\Bundle\\IcecatConnectorBundle\\Tests\\": "tests/Integration/",
```

Comment:
I would have put Icecat tests in a `tests/Icecat/Integration` to respect the same hierarchy than `src`.

Just a proposition, will not block the merge.